content/internal-docs/creating-stream-9-repos.md

```diff
@@ -0,0 +1,66 @@
+---
+title: "Creating CentOS Stream 9 source-git repositories"
+draft: false
+weight: 1
+---
+
+# Creating CentOS Stream 9 source-git repositories
+
+Source-git repos for CentOS Stream 9 are stored as separate repositories in
+https://gitlab.com/redhat/centos-stream/src on a branch `c9s`.
+
+Here is how the repositories should be set up:
+
+* Issues are disabled (BZ is the official bug tracker).
+
+* Merge requests are enabled.
+
+* The repositories are open to public (for contributions).
+
+* Description is set:
+  ```
+  Source repo for CentOS Stream package $package.
+  You can contribute here by following https://wiki.centos.org/Contribute/CentOSStream/
+  ```
+
+* `c9s` branch is protected, though force-pushes are allowed from maintainers (for sake of rebasing).
+
+
+## How to create such repo
+
+[using `packit init`]({{< ref "init.md" >}})
+
+
+## Pushing the repository to GitLab
+
+We have an identity,
+[centos-stream-packit](https://gitlab.com/centos-stream-packit), which has
+permissions to create new repositories in the namespace.
+
+Here comes a sample script utilizing ogr and python-gitlab to create a new repo
+to satisfy the requirements above:
+```
+import ogr
+import os
+
+TARGET_GROUP = "redhat/centos-stream/src"
+GITLAB_TOKEN = os.getenv("GITLAB_TOKEN")
+
+gitlab_service = ogr.GitlabService(GITLAB_TOKEN)
```

Comment:
I don't see `gitlab_service` used anywhere

Reply:
that was a naive copy-pasta-edit-hope-for-da-best :) will drop that line

there is actually more to it